OracleTuxedo/art_runtime/README.md

```diff
@@ -0,0 +1,35 @@
+# Introduction
+This folder contains the information and examples of how to use [Tuxedo ART Runtime](http://docs.oracle.com/cd/E72452_01/artrt/docs1222/index.html) with [Docker](https://www.docker.com/).
+
+## Contents
+It is based on the Oracle Tuxedo image.
+
+##Prerequisite
+1. Use btrfs with docker:
+   With big image size, it's better to use btrfs with docker, please follow the link below to set Docker using btrfs.
+   https://docs.docker.com/engine/userguide/storagedriver/btrfs-driver/#configure-btrfs-on-sles
+
+
+## To use
+Pre-installation:
+1. Download the binaries and copy them to `pwd`/bin
```

Comment:
Why do you want the binaries copied into `pwd`/bin? Why not put them in the same folder as the `Dockerfile` just like every other image in this repository?